manifest.yml

```diff
@@ -4,5 +4,6 @@ applications:
   memory: 512M
   host: collection-instrument-demo
   services:
-    - ras-postgres
+    - rm-pg-db
```

Comment:
If it's the name of the CF database service, then it should be unique to ras-collection-instrument, e.g. ras-ci-db